sdk/metrics-testing/src/main/java/io/opentelemetry/sdk/testing/assertj/metrics/package-info.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+@ParametersAreNonnullByDefault
+package io.opentelemetry.sdk.testing.assertj.metrics;
+
+import javax.annotation.ParametersAreNonnullByDefault;
```

Comment:
This was missing from the `io.opentelemetry.sdk.testing.assertj.metrics` package. 

On a related note, `io.opentelemetry.sdk.testing.assertj.metrics` should remove the `metrics` suffix to match tracing and minimize impact when these `:sdk:metrics-testing` eventually merges with `:sdk:testing`.